tests/spec/w3c/headers-spec.js

```diff
@@ -221,6 +221,18 @@ describe("W3C — Headers", function() {
     });
   });
 
+  describe("title", () => {
+    it("uses <h1> if already present", async () => {
+      const ops = makeStandardOps();
+      ops.body = "<h1 id='title'>Hello <code>World</code></h1>" + makeDefaultBody();
+      const doc = await makeRSDoc(ops);
+      expect($(".head .title.p-name", doc).length).toEqual(1);
+      expect($(".title.p-name", doc).length).toEqual(1); // original <h1> is removed
+      expect($("#title code", doc).length).toEqual(1); // html is not escaped
```

Comment:
Previously whenever I had written a test, I remember @marcoscaceres suggesting me to use a variable whenever a need of writing out comments at the end of statements is felt.

This actually improves the readability of the code significantly for any third person viewing . Otherwise this patch looks good :+1: . Good work :smile:

Reply:
I didn't get that quite clearly. Can you suggest a variable?

Reply:
Yep sure. Checkout this link https://github.com/w3c/respec/pull/1487/files

Though initially, I had used `expect` statements to compare with the `output` (toEqual() method) directly, it wasn't quite obvious for a new reader to comprehend that, without again and again keeping an eye on it. 
Making explanatory variables would now enable you to understand this testing code quickly. Same for someone who wants to get over with the code without wasting too much time. :sweat_smile: 

I hope you are getting my point. These are just some good practices I picked up along the way.